client/my-sites/themes/single-site-wpcom.jsx

```diff
@@ -10,32 +11,72 @@ import CurrentTheme from 'my-sites/themes/current-theme';
 import SidebarNavigation from 'my-sites/sidebar-navigation';
 import ThanksModal from 'my-sites/themes/thanks-modal';
 import { connectOptions } from './theme-options';
+import { addTracking } from './helpers';
 import { FEATURE_ADVANCED_DESIGN } from 'lib/plans/constants';
 import UpgradeNudge from 'my-sites/upgrade-nudge';
 import QuerySitePlans from 'components/data/query-site-plans';
 import QuerySitePurchases from 'components/data/query-site-purchases';
 import ThemeShowcase from './theme-showcase';
+import ThemesSelection from './themes-selection';
 
-export default connectOptions(
-	( props ) => {
-		const { site, siteId, translate } = props;
+const SingleSiteThemeShowcaseWpcom = React.createClass( {
+	propTypes: {
+		getScreenshotOption: PropTypes.func,
+	},
+
+	render() {
```

Comment:
Okay, as discussed on Slack, it seems this commit would get us to step 2. However, don't use `React.createClass` here -- it's deprecated, and we don't need it here. Just keep the functional component as-is; and I think you can even keep the `connectOptions` wrapping in this place. (Essentially, leave those lines above unchanged 😉 , except for the `const` definition, where you'll just add the ones you need.)